test/acceptance/page-objects/claimJourney-page.js

```diff
@@ -70,16 +71,18 @@ class StartPageActions extends CommonActions {
   }
 
   async validData(){
-    await this.inputValidCrn(process.env.CRN_USERNAME)
+    const sleep = (waitTimeInMs) => new Promise(resolve => setTimeout(resolve, waitTimeInMs))
+    await sleep(5000)
+    await this.inputValidCrn(process.env.CRN_CLAIM)
     await this.inputPassword(process.env.CRN_PASSWORD)
+    console.log(`################ CRN is ${process.env.CRN_CLAIM}`)
+    console.log(`################# PASSWORD is ${process.env.CRN_PASSWORD}`)
```

Comment:
Are these for logging or should they be removed?

Reply:
meant for debugging. However, I will remove
Thanks